packages/now-next/src/index.ts

```diff
@@ -500,40 +511,17 @@ export const build = async ({
           headers: {
             'cache-control': `public,max-age=${MAX_AGE_ONE_YEAR},immutable`,
           },
-          continue: true,
         },
-        {
-          src: path.join('/', entryDirectory, '_next(?!/data(?:/|$))(?:/.*)?'),
-        },
-        // Next.js pages, `static/` folder, reserved assets, and `public/`
-        // folder
-        { handle: 'filesystem' },
 
-        // This needs to come directly after handle: filesystem to make sure to
-        // 404 and clear the cache header for _next requests
-        {
-          src: path.join(
```

Comment:
We still need this after handle: miss

Reply:
Since it's only applied during `handle: hit` it would never need to be cleared in `handle: miss` since it can't `miss` after `hit` right?

Reply:
This route isn't just for that. It's so you can't rewrite /_next

Reply:
If they rewrite `/_next` and it's a `hit` wouldn't it still not call `handle: miss` and if it doesn't `hit` it would never get the header and just 404 anyways right?

Reply:
No handle miss is called before rewrites

Reply:
I see, updated. We currently throw an error if the route doesn't have `continue: true` after `handle: 'miss'` it will still 404 correctly right?

Reply:
We can use check: true with dest: $0

Reply:
Updated